ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -67,7 +61,9 @@ public static void updateEth1Data(BeaconState state) throws Exception {
         if (eth1_data_vote
                 .getVote_count()
                 .times(UnsignedLong.valueOf(2))
-                .compareTo(UnsignedLong.valueOf(Constants.ETH1_DATA_VOTING_PERIOD * EPOCH_LENGTH))
+                .compareTo(
+                    UnsignedLong.valueOf(
+                        Constants.ETH1_DATA_VOTING_PERIOD * Constants.EPOCH_LENGTH))
```

Comment:
EPOCH_LENGTH is already imported staticly, so it is not necessary to prepend `Constants.`

Reply:
I actually had removed the importing of any constant in this PR.